storage/build.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE.md for licensing terms.
+
+use semver::Version;
+
+const VERSION: &str = env!("CARGO_PKG_VERSION");
+
+fn main() {
+    let crate_version = Version::parse(VERSION).expect("failed to parse CARGO_PKG_VERSION");
+
+    // Set FIREWOOD_DB_VERSION to the major version of the crate. E.g., if the version
```

Comment:
:trophy: on approach

Let's move this comment to the top of the function or as a doc comment for the whole file so I don't have to read the function to figure out what it does.
```suggestion
    // Put FIREWOOD_DB_VERSION into the build environment
    // this is the major version of semver release
```